x-pack/test/api_integration/apis/uptime/graphql/monitor_states.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { monitorStatesQueryString } from '../../../../../legacy/plugins/uptime/public/queries/monitor_states_query';
+import monitorStates from './fixtures/monitor_states.json';
+import monitorStatesIdFiltered from './fixtures/monitor_states_id_filtered.json';
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: { getService: any }) {
+  describe('monitorStates', () => {
+    const supertest = getService('supertest');
+
+    it('will fetch monitor state data for the given date range', async () => {
+      const getMonitorStatesQuery = {
+        operationName: 'MonitorStates',
+        query: monitorStatesQueryString,
+        variables: {
+          dateRangeStart: '2019-01-28T17:40:08.078Z',
+          dateRangeEnd: '2019-01-28T19:00:16.078Z',
+        },
+      };
+
+      const {
+        body: { data },
+      } = await supertest
+        .post('/api/uptime/graphql')
+        .set('kbn-xsrf', 'foo')
+        .send({ ...getMonitorStatesQuery });
+
+      expect(data).to.eql(monitorStates);
```

Comment:
This test is failing in CI because the timestamps are still "2019" 🤔

Reply:
Yes I'm investigating now. Rather not good.

Reply:
Fixed in https://github.com/elastic/kibana/pull/40217/commits/31439818b61bbc910ecaf1cf5e6a9fad71cdb392 and https://github.com/elastic/kibana/pull/40217/commits/e43db48527e833cc01368b86f690ecf41073a78d.